helm/exporter-node/templates/psp-clusterrole.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.global.pspEnable }}
```

Comment:
Rbac should be required to create ClusterRole and ClusterRoleBinding please add the line below in all resource added by you

``` {{- if .Values.global.rbacEnable }} ```